CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Full documentation for RPP is available at [https://rocm.docs.amd.com/projects/rpp/en/latest](https://rocm.docs.amd.com/projects/rpp/en/latest)
  
+## RPP 1.9.2 (Unreleased)
+
+### Changes
```

Comment:
@kiritigowda @rrawther @LakshmiKumar23 For every new functionality addition PR issued, we are now incrementing the version like this - 1.9.1 -> 1.9.2 (for melfilter bank) -> 1.9.3 (for spectrogram) -> 1.9.4 (for box filter).

Please see in the 3 open PRs - does this work?
The versioning is uniformly changed in 3 RPP files in every PR:
- CHANGELOG.md
- CMakeLists.txt
- include/rpp_version.h

Reply:
@r-abishek RPP_VERSION_PATCH should be good. We can increment MINOR for a whole new functionality like Audio